src/Pages/Profile/MemberView/InfoCard.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import { Input } from 'reactstrap'
+import './ProfileModifier.css'
+
+export default function ProfileCard (props) {
+  const [month] = useState(parseInt(new Date().getMonth()) + 1)
+  const [pic] = useState(getPic())
+
+  function getPic () {
+    let pic
+    if (month === 3 || month === 4 || month === 5) { pic = require('./Image/spring.jpg') } else if (month === 6 || month === 7 || month === 8) { pic = require('./Image/summer.jpeg') } else if (month === 9 || month === 10 || month === 11) { pic = require('./Image/fall.jpeg') } else if (month === 12 || month === 1 || month === 2) { pic = require('./Image/winter.jpeg') }
```

Comment:
None of this!
Instead, make another `.js` file in the `MemberView/` directory that will have a function like below:
```js
export default function getPicBySeason() {
  // code here
}
```

Then, just import it in this file and call on it. 

A suggestion for this exported function's body:
```js
import fall from './Image/fall.jpeg'
// import the rest of the seasons like above


export default function getPicBySeason() {
  let pic = null
  if (month === 9 || month === 10 || month === 11) {
    pic = fall 
  } // else if (...)
  
  return pic
}

Reply:
By the way, `getMonth` is [indexed at zero](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth), keep that in mind